pandas/conftest.py

```diff
@@ -1692,6 +1693,22 @@ def any_skipna_inferred_dtype(request):
     return inferred_dtype, values
 
 
+# ----------------------------------------------------------------
+# Ctypes
+# ----------------------------------------------------------------
+@pytest.fixture
+def mock_ctypes(monkeypatch):
+    """
+    Mocks WinError to help with testing the clipboard.
+    """
+
+    def _mock_win_error():
+        return "Window Error"
+
+    # Set raising to False because WinError won't exist on non-windows platforms
+    monkeypatch.setattr("ctypes.WinError", _mock_win_error, raising=False)
```

Comment:
Just to be safe, could you use the context manger version?

```
with monkeypatch.context() as m:
    m.setattr(...)
    yield
```

Reply:
Updated